codecov.yml

```diff
@@ -0,0 +1,6 @@
+comment: off
+coverage:
+  status:
+    project:
+      default:
+        target: auto
```

Comment:
Traditionally (also by default), we would send a failure for the coverage context if there was any decrease.